README.md

```diff
@@ -95,44 +95,35 @@ It is possible for properties to collide, between both stamps, and between diffe
 
 **Different descriptor properties, one or more stamps:**
 
-* Optionally warn on collision at stamp creation time (set by configuration, off by default)
 * Instance properties have lowest priority
 * Shallow properties override deep properties
 * Descriptors override everything
 
 #### Configuration
 
-Stamp composition and instance creation behaviors can be manipulated by configuration stamps:
+Stamp composition and instance creation behaviors can be manipulated by configuration stamps. For instance, it's possible to create [a stamp that warns on collisions](https://github.com/stampit-org/collision-stamp) across different descriptor properties. e.g.:
 
 ```js
+import warnOnCollisions from 'collision-stamp';
```

Comment:
Before the code I think we should state: "Configuration example"

Also, add maybe note that configuration features are up to implementers.